SmartDeviceLink/SDLRPCRequestFactory.m

```diff
@@ -393,6 +398,15 @@ + (SDLDeviceInfo *)sdl_buildDeviceInfo {
     return deviceInfo;
 }
 
++ (SDLAppInfo*)sdl_buildAppInfo {
+    SDLAppInfo* appInfo = [[SDLAppInfo alloc] init];
+    NSBundle* mainBundle = [NSBundle mainBundle];
+    NSDictionary* bundleDictionary = mainBundle.infoDictionary;
+    appInfo.appVersion = [bundleDictionary objectForKey:kBundleShortVersionStringKey];
```

Comment:
This can use shortened syntax: `bundleDictionary[kBundleShortVersionStringKey]`.